content/articles/2019-04-r-version-support.Rmarkdown

```diff
@@ -0,0 +1,20 @@
+---
+title: R version support
+author: Mara Averick
+date: '2019-04-01'
+slug: r-version-support
+description: >
+    Which versions of R do tidyverse packages support?
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/GSCtoEEqntQ
+  author: Ralph Howald
+---
+
+## Which versions of R do tidyverse packages support? 
+
+Though package-specific minimum versions of R are given in the `Depends` fields of individual `DESCRIPTION` files, our general policy is to support the __current version__, the __devel version__, and __four previous versions__ of R.
+
+With the [prerelease of R version 3.6.0 (Planting of a Tree)](https://cran.r-project.org/src/base-prerelease/) already available, and its final release slated for 2019-04-26, you can expect tidyverse packages to drop support for R 3.1 in the coming weeks. The most recent version of [highr](https://CRAN.R-project.org/package=highr), which is used by the [R Markdown](https://rmarkdown.rstudio.com/) package, requires R ≥ 3.2.3.
```

Comment:
Needs a segue before last sentence. Maybe something like: "This aligns R version support in the tidyverse with RMarkdown, which as of the latest release requires R 3.2.3 (due to an update to the highr package).